packages/cursorless-engine/src/actions/GetTargets.ts

```diff
@@ -0,0 +1,17 @@
+import { Target } from "../typings/target.types";
+import { ActionReturnValue, SimpleAction } from "./actions.types";
+
+export default class GetTargets implements SimpleAction {
+  constructor() {
+    this.run = this.run.bind(this);
+  }
+
+  async run(targets: Target[]): Promise<ActionReturnValue> {
```

Comment:
I think a flash would be nice. WDYT @phillco @dyamito

Reply:
You mean a visual flash?

Reply:
I left it out since the sidecar wouldnt benefit by flashes in vscode.

Reply:
I would agree. I also don't know if flashing is synchronous but we wouldn't want to have any delay here.

Reply:
Good call